### PR TITLE
Add new keyboard column13 series

### PR DESCRIPTION
**Pull request by abplus-lab**
## Description

Add the following 3 types of keyboards

* Column13 ansi
* Column13 iso
* Column13 ortho

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

None

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — COMMENTED** [review]
Please revert the changes to the submodules. 

You may need to run `make git-submodule`

**Comment by kazhida** [comment]
Than you for advice.
I corrected sub-modules problem .

**Review by drashna — APPROVED** [review]

**Review by tzarc — CHANGES_REQUESTED** [review]
Please remove any changes not related to `keyboards/column13` -- there are a couple of modified files in other keyboards that should not be present.

As there is no difference between implementations of `matrix.c`, please move all of the files in `ansi` and `iso` up one directory into `column13`, and provide `LAYOUT_ansi` and `LAYOUT_iso` macros, with corresponding keymaps. As far as I can see, there's no reason to define these as two different keyboards.

File: keyboards/column13/ansi/config.h [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+/* disable these deprecated features by default */
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
+
+/* Bootmagic Lite key configuration */
+//#define BOOTMAGIC_LITE_ROW 0
+//#define BOOTMAGIC_LITE_COLUMN 0
+
+#define DEBUG_ACTION 1
+#define USE_I2C 1
```
Comment (drashna) [inline keyboards/column13/ansi/config.h:155]:
```suggestion
```

File: keyboards/column13/ansi/matrix.c [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+bool matrix_scan_custom(matrix_row_t current_matrix[]) {
+    bool changed = false;
+
+    dprint("matrix_scan_custom");
+    for (uint8_t i = 0; i < MATRIX_ROWS; i++) {
+        matrix_row_t last_row = current_matrix[i];
+        matrix_row_t current_row = expander_readPins(&expanders[i]);
+        if (current_row != 0 && current_row != last_row) {
+            xprintf("row[%d]: %04X -> %04X\n", i, last_row, current_row);
```
Comment (drashna) [inline keyboards/column13/ansi/matrix.c:56]:
```suggestion
            dprintf("row[%d]: %04X -> %04X\n", i, last_row, current_row);
```

File: keyboards/column13/iso/config.h [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+/* disable these deprecated features by default */
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
+
+/* Bootmagic Lite key configuration */
+//#define BOOTMAGIC_LITE_ROW 0
+//#define BOOTMAGIC_LITE_COLUMN 0
+
+#define DEBUG_ACTION 1
+#define USE_I2C 1
```
Comment (drashna) [inline keyboards/column13/iso/config.h:155]:
```suggestion
```

File: keyboards/column13/iso/matrix.c [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+bool matrix_scan_custom(matrix_row_t current_matrix[]) {
+    bool changed = false;
+
+    dprint("matrix_scan_custom");
+    for (uint8_t i = 0; i < MATRIX_ROWS; i++) {
+        matrix_row_t last_row = current_matrix[i];
+        matrix_row_t current_row = expander_readPins(&expanders[i]);
+        if (current_row != 0 && current_row != last_row) {
+            xprintf("row[%d]: %04X -> %04X\n", i, last_row, current_row);
```
Comment (drashna) [inline keyboards/column13/iso/matrix.c:56]:
```suggestion
            dprintf("row[%d]: %04X -> %04X\n", i, last_row, current_row);
```

File: keyboards/column13/modulo/expander.c [hunk 5]
```diff
@@ -0,0 +1,69 @@
+//
```
Comment (drashna) [inline keyboards/column13/modulo/expander.c:1]: This appears to be a re-implementation of https://github.com/qmk/qmk_firmware/blob/master/drivers/gpio/pca9555.c

If so, please use that instead.

File: keyboards/column13/ortho/config.h [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+/* disable these deprecated features by default */
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
+
+/* Bootmagic Lite key configuration */
+//#define BOOTMAGIC_LITE_ROW 0
+//#define BOOTMAGIC_LITE_COLUMN 0
+
+#define DEBUG_ACTION 1
+#define USE_I2C 1
```
Comment (drashna) [inline keyboards/column13/ortho/config.h:155]:
```suggestion
```

File: keyboards/pkb65/info.json [hunk 7]
```diff
@@ -1,77 +1,77 @@
...
-            {"label":"Alt", "x":2.5, "y":4, "w":1.25}, 
-            {"x":3.75, "y":4, "w":6.25}, 
-            {"label":"Alt", "x":10, "y":4, "w":1.25}, 
-            {"label":"Fn", "x":11.25, "y":4, "w":1.25}, 
-            {"label":"\u2190", "x":13, "y":4}, 
-            {"label":"\u2193", "x":14, "y":4}, 
-            {"label":"\u2192", "x":15, "y":4}]
-      }
-  }
+{
```
Comment (drashna) [inline keyboards/pkb65/info.json:1]: This change should be reverted.

File: keyboards/column13/ortho/matrix.c [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+bool matrix_scan_custom(matrix_row_t current_matrix[]) {
+    bool changed = false;
+
+    dprint("matrix_scan_custom");
+    for (uint8_t i = 0; i < MATRIX_ROWS; i++) {
+        matrix_row_t last_row = current_matrix[i];
+        matrix_row_t current_row = expander_readPins(&expanders[i]);
+        if (current_row != 0 && current_row != last_row) {
+            xprintf("row[%d]: %04X -> %04X\n", i, last_row, current_row);
```
Comment (drashna) [inline keyboards/column13/ortho/matrix.c:56]:
```suggestion
            dprintf("row[%d]: %04X -> %04X\n", i, last_row, current_row);
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**